tests/hotplug/cpu.go

```diff
@@ -0,0 +1,194 @@
+package hotplug
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"kubevirt.io/kubevirt/tests/flags"
+	util2 "kubevirt.io/kubevirt/tests/util"
+	"time"
+
+	"kubevirt.io/kubevirt/pkg/pointer"
+	"kubevirt.io/kubevirt/tests/libvmi"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
+	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/libwait"
+)
+
+var _ = Describe("[sig-compute][serial]CPU Hotplug", decorators.SigCompute, decorators.RequiresTwoSchedulableNodes, func() {
+	var virtClient kubecli.KubevirtClient
+	var originalKv *v1.KubeVirt
+
+	BeforeEach(func() {
+		virtClient = kubevirt.Client()
+		originalKv = util2.GetCurrentKv(virtClient)
+		updateStrategy := &v1.KubeVirtWorkloadUpdateStrategy{
+			WorkloadUpdateMethods: []v1.WorkloadUpdateMethod{v1.WorkloadUpdateMethodLiveMigrate},
+		}
+		patchWorkloadUpdateMethod(originalKv.Name, virtClient, updateStrategy)
+
+	})
+
+	AfterEach(func() {
+		patchWorkloadUpdateMethod(originalKv.Name, virtClient, &originalKv.Spec.WorkloadUpdateStrategy)
+	})
```

Comment:
I think this isn't necessary and happens automatically

Reply:
AFAIK what is restored automatically is `spec.Configuration` but here I change `spec.workloadUpdateStrategy`

Reply:
I see that we replace the spec, I will remove the restore code then! thank you!